code/modules/research/designs/medical_designs.dm

```diff
@@ -498,7 +498,7 @@
 
 /datum/design/cyberimp_nutriment
 	name = "Nutriment Pump Implant"
-	desc = "This implant with synthesize and pump into your bloodstream a small amount of nutriment when you are starving."
+	desc = "This implant will synthesize and pump into your bloodstream a small amount of nutriment when you are starving."
```

Comment:
```suggestion
	desc = "When you're starving, this implant will synthesize a small amount of nutriment and pump it into your bloodstream."
```
The sentence as a whole is a little weird, maybe change it to smth like this? Up to you, though

Reply:
honestly that is very good, I like it